src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -36,7 +36,7 @@ const Dashboard = props => {
           totalDisabilityRating={props.totalDisabilityRating}
         />
       )}
-      <div className="vads-l-grid-container vads-u-padding-x--0">
+      <div className="vads-l-grid-container large-screen:vads-u-padding-x--0">
```

Comment:
I wonder if we want the padding even on large screens. I think the Profile has the standard padding. And the name tag badge does not left align to the main content if there is no padding.

Reply:
There are lots of little CSS issues that aren't worth addressing yet, but that is one of them.

Reply:
I removed this class :) No more `vads-u-padding-x--0`